www_src/pages/element/text-editor.jsx

```diff
@@ -16,9 +16,16 @@ var TextEditor = React.createClass({
       fontWeight: 'normal',
       fontStyle: 'normal',
       textDecoration: 'none',
-      textAlign: 'center'
+      textAlign: 'center',
+      innerHTML: 'Hello world'
     };
   },
+  editText: function () {
```

Comment:
this doesn't work in Android's WebView (no `alert`, `prompt`, and `confirm` support) unless we switch over to WebChromeView for the moment (https://github.com/mozilla/webmaker-app/pull/1682)